src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -90,81 +90,101 @@ class SpfxProjectRenameCommand extends BaseProjectCommand {
       }
     }
 
-    const filePaths: string[] = [
-      path.join(this.projectRootPath, 'package.json'),
-      path.join(this.projectRootPath, '.yo-rc.json'),
-      path.join(this.projectRootPath, 'config/package-solution.json'),
-      path.join(this.projectRootPath, 'config/deploy-azure-storage.json'),
-      path.join(this.projectRootPath, 'README.md')
-    ];
+    const replacePackageJsonContent = (filePath: string) => {
```

Comment:
Let's move these functions outside of `action` to a separate class-level method.

Reply:
Hi @waldekmastykarz! Shall I create a single class method that contains all these functions or do each of these functions need to be written as separate class methods?

Reply:
Let's do a separate class method for each modification. As the project evolves and new elements are added it will be easier to keep track of the different pieces that need to be renamed.